projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import Card from 'components/card';
+import React, { useState, useCallback, useMemo } from 'react';
+import { connect } from 'react-redux';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getStatsData,
+	getPluginItems,
+	isFetchingPluginsData,
+	isFetchingStatsData,
+} from 'state/at-a-glance';
+import RecordMeterBar from 'components/record-meter-bar';
+
+import { Popup } from './popup';
+
+import './style.scss';
+
+const BackupUpgrade = ( { comments, isFetchingData, plugins, posts } ) => {
+	const [ showPopup, setShowPopup ] = useState( true );
+
+	const onClosePopup = useCallback( () => setShowPopup( false ), [] );
+
+	const items = useMemo( () => {
+		return [
+			{ count: posts, label: __( 'Posts', 'jetpack' ), backgroundColor: '#00BA37' },
+			{ count: plugins, label: __( 'Plugins', 'jetpack' ), backgroundColor: '#3895BA' },
+			{ count: comments, label: __( 'Comments', 'jetpack' ), backgroundColor: '#E68B28' },
+		];
+	}, [ comments, plugins, posts ] );
+
+	return (
+		! isFetchingData &&
+		showPopup &&
+		posts > 20 && (
```

Comment:
Stylistically, I'd like to see us put `20` in a constant at the top of the file so that future engineers can come back and easily understand what it means. In this case, I'm guessing it would stand for something like `MIN_POSTS_FOR_VISIBLE_BAR`?

Reply:
Yeah, good suggestion. Thank you.